### ISSUE TITLE
Добавить на фронтенде пользовательские настройки

### ISSUE THREAD

**Issue by Linchuki** [hexlet-rus/runit#502]
### Problem description

Добавляются настройки для переключения темы и локали. Такие вещи хорошо настраивать в профиле пользователя
Task #369 разбили на 2 части

### Proposed solution

Добавить в настройки профиля (скрыть через фича тогглер) https://runit.hexlet.ru/settings

выбор темы и локали
Тема - светлая, темная, локаль - ru, en

**Comment by bobrov-site** [comment]
могу взять

**Comment by bobrov-site** [comment]
У меня вопрос реализации: 

Правильно понял, что мы добавляем отдельный подраздел, например называем его внешний вид. Ниже будет так же дропдауном как и в шапке возможность выбора темы и языка. После нажатия на кнопку изменить, у нас сохраняются изменения и меняется состояние так же в шапке. При перегрузки страницы мы просто подтягиваем данные из бэка. Верно?

Схематичный скриншот реализации

<img width="1435" alt="Снимок экрана 2024-06-07 в 16 48 26" src="https://github.com/hexlet-rus/runit/assets/39225852/f1a21c9b-455b-46b7-a6ca-348ebf4c8c3a">

**Comment by fey** [comment]
а щас на фронте как это сделано? По идее да. пользователь зайдет с любого устройства и у него подтянутся данные с бекенда

**Comment by bobrov-site** [comment]
@fey на фронте так и сделано, только без нового раздела внешний вид. В целом, если все так, то готов выполнять.

**Comment by fey** [comment]
Ххмхмхм, с одной стороны мы дублируем настройки с другой...  Давайте пока не делать, нам по идее надо сделать привязку-отвязку аккаунта аккаунта Github, плюс удаление аккаунта (на бекенде мб этого и нет). Можно пока на фронте заглушки делать. А для смены вида у нас есть сверху переключалки, пока ими обойдемся.

**Comment by fey** [comment]
так, отбой предыдущего отбоя. Давайте добавлять все таки настройки внешнего вида) Тут же еще расцветка редактора может быть.

**Comment by bobrov-site** [comment]
@fey добавил пуллреквест. Могу в него так же добавить возможность переключения темной или светлой темы редактора кода из панели настроек пользователя.

**Comment by bobrov-site** [comment]
@fey вопрос по поводу расцветки редактора. Сейчас у нас он подвязан к теме приложения. Не будет ли странно, если тема сайта будет светлая, а тема редактора темная. И наоборот. В общем хотелось бы понять, действительно ли стоит добавлять расцветку так же для редактора кода.
